packages/kbn-ui-shared-deps-npm/webpack.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = (_, argv) => {
     externals: {
       module: 'module',
     },
-    mode: process.env.NODE_ENV || 'development',
+    mode: 'production',
```

Comment:
@clintandrewhall can you explain this a bit? What is this doing?

Reply:
I thought the flag was dependent on being in production.  That may have changed.  @cee-chen ?

Reply:
Sorry, not sure I have context for this question - if it's around `setEuiDevProviderWarning`, it doesn't check for the current node env whatsoever, it just cares that the global provider warning variable was set.